packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -472,6 +474,17 @@ const Summary: React.FunctionComponent = () => {
 
 	const hasPaidPlan = site.isPaidPlan;
 
+	const onAskForHelpClick = ( event: React.MouseEvent< HTMLAnchorElement, MouseEvent > ) => {
+		const helpHref = ( event.target as HTMLAnchorElement ).getAttribute( 'href' );
+
+		if ( ! helpHref ) {
+			return;
+		}
+
+		redirectTo( helpHref );
```

Comment:
When I click on the "ask for help button" I kinda expect to be redirected to WordPress chat instead of the [help page](https://wordpress.com/help). Is this correct?

Reply:
The specs defined in #47395 read:

_"The link reads "Ask a Happiness Engineer" and, when clicked, should redirect the user to the /help page"_

Just double-checking with @razvanpapadopol that this is still the expected beheviour

Reply:
When I'm in the editor and I wantto contact HE, I press _Contact Us_ button on this pop-up:
<img width="371" alt="Screenshot 2020-11-24 at 19 01 44" src="https://user-images.githubusercontent.com/14192054/100127258-99291080-2e87-11eb-9535-5555380d3d7d.png">

Then chat is opened. But I guess this is not applicable for Free & Personal plans. However, I'm sure there is a component in Calypso with all this logic.
<img width="351" alt="Screenshot 2020-11-24 at 19 02 02" src="https://user-images.githubusercontent.com/14192054/100127275-9c240100-2e87-11eb-8f5c-ccc866959e70.png">

@ollierozdarz is this something we want to integrate also in Focused Launch?

Reply:
@razvanpapadopol I think using the functionality of the component in the last screenshot (but maybe without the ( ? ) icon though, just showing the widget via clicking on the 'Ask a happiness engineer') would be a great solution.

Reply:
Thank you for your answer! I opened #47784 to track this task — we can continue the conversation there